strawberryfields/ops.py

```diff
@@ -1424,16 +1424,18 @@ class PassiveChannel(Channel):
         Acts the following transformation on the state:
 
         .. math::
-            a^{\dagger}_i \to \sum_j T_{ij} a^{\dagger}j
+            a^{\dagger}_i \to \sum_j T_{ij} a^{\dagger}_j
+
     """
 
     def __init__(self, T):
+        T = np.atleast_2d(T)
         super().__init__([T])
         self.ns = T.shape[0]
 
     def _apply(self, reg, backend, **kwargs):
         p = par_evaluate(self.p)
-        backend.passive(p[0], *reg)
+        backend.passive(p[0], reg)
```

Comment:
was this a bug?

Reply:
no, but i realised that i didn't need to unpack, so i removed it here and in the backend function too